scipy/stats/stats.py

```diff
@@ -6637,31 +6637,23 @@ def _count_paths_outside_method(m, n, g, h):
     # B is an array just holding a few values of B(x,y), the ones needed.
     # B[j] == B(x_j, j)
     if lxj == 0:
-        return np.round(special.binom(m + n, n))
+        return special.binom(m + n, n)
     B = np.zeros(lxj)
     B[0] = 1
     # Compute the B(x, y) terms
-    # The binomial coefficient is an integer, but special.binom() may return a float.
-    # Round it to the nearest integer.
     for j in range(1, lxj):
-        Bj = np.round(special.binom(xj[j] + j, j))
-        if not np.isfinite(Bj):
-            raise FloatingPointError()
+        Bj = special.binom(xj[j] + j, j)
```

Comment:
If FloatingPointError is not explicitly raised, the the docstring for `_count_paths_outside_method` needs to be updated to reflect that.

Reply:
Okay, should the section just be deleted, or should be mentioned that the function is wrapped with `np.errstate`?

Reply:
If `_count_paths_outside_method` doesn't raise FloatingPointError, just remove it from the docstring.  How other functions call it or wrap it doesn't need to get mentioned in its docstring.